sklearn/model_selection/_search.py

```diff
@@ -1411,7 +1411,24 @@ class RandomizedSearchCV(BaseSearchCV):
         A generator over parameter settings, constructed from
         param_distributions.
 
+
+    Examples
+    --------
+    >>> from sklearn import linear_model, datasets
+    >>> from sklearn.model_selection import RandomizedSearchCV
+    >>> from scipy.stats import  uniform
+    >>> iris = datasets.load_iris()
+    >>> logistic = linear_model.LogisticRegression(multi_class='auto',
```

Comment:
`multi_class='auto'` and `penalty='l2'` are the default already. No need to specify them. 
```suggestion
    >>> logistic = LogisticRegression(max_iter=200)
```